autopush/message/handler.py

```diff
@@ -0,0 +1,42 @@
+from twisted.internet.threads import deferToThread
+
+from autopush.base import (
+    threaded_validate,
+    BaseWebHandler
+)
+from autopush.message.validation import MessageSchema
+from autopush.utils import WebPushNotification
+
+
+class MessageHandler(BaseWebHandler):
```

Comment:
Why isn't this going to live in autopush/web? I'm trying to understand the architecture consistency being chosen. The intent with autopush/web was all web handlers would live there, like the web/simplepush, web/webpush, web/registration, web/message, etc. Now it seems the web 'base' is living in both autopush/base.py and autopush/web/base. autopush/base.py was originally only a refactor while endpoint.py lives, ideally it was going to then die so that web/base would contain the base logic for all web handlers.

If the new intent is that all the web handling lives in its own package off the root, we'll need to refactor the refactored webpush/simplepush again for consistent architecture.